stl/inc/algorithm

```diff
@@ -6628,6 +6628,154 @@ void sort(_ExPo&& _Exec, const _RanIt _First, const _RanIt _Last) noexcept /* te
     // order [_First, _Last)
     _STD sort(_STD forward<_ExPo>(_Exec), _First, _Last, less{});
 }
+
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    template <class _It, class _Pr, class _Pj>
+    constexpr void _Insertion_sort_common(const _It _First, const _It _Last, _Pr _Pred, _Pj _Proj) {
+        // insertion sort [_First, _Last)
+        _STL_INTERNAL_STATIC_ASSERT(bidirectional_iterator<_It>);

```

Comment:
Recently the convention was to constrain public helper functions rather than using asserts

Reply:
I'll constrain these.